content/en/docs/concepts/scheduling/configure-scheduler.md

```diff
@@ -0,0 +1,223 @@
+
+* [How does the Kubernetes scheduler work?](docs/concepts/scheduling/how-scheduler-works/)
+* [How to customize and extend the Kubernetes scheduler?](/docs/concepts/scheduling/customization-and-extension/)
+* How to configure your Kubernetes scheduler?
+* [Choose scheduling algorithms and strategies for your cluster.](TODO)
+* [Learn the kube-scheduler's design and implementation from source code.](TODO)
+
+This page explains how to configure Kubernetes scheduler.
+
+{{% /capture %}}
+
+{{% capture body %}}
+
+## Scheduler
+
+The [Kubernetes scheduler](/docs/reference/command-line-tools-reference/kube-scheduler/) is a main component of Kubernetes. It is responsible for selecting the best node for the pod based on a series of predefined scheduler policies. Kubenetes provides the default scheduler and policies to place the pod on the appropriate node in a Kubernetes cluster.
+
+## Configure the Policies
+
+The Kubernetes scheduler plugin frameworks provides the user with the ability to:
+
+1. Customize the default scheduler policies. 
+2. Extend the scheduler.
+3. Write the new scheduler(s) to either run alongside the default scheduler or replace the default scheduler.
+
+For more information, see the article [Customization and Extension ](/docs/concepts/scheduling/customization-and-extension/).
+
+The default scheduler policies fit most use cases. Users can fine-tune the scheduler behavior without modifying the default scheduler by specifying the customized-policy file when the kube-scheduler starts with the `--policy-config-file` option.
+
+Given below is an example of a typical policy file:
+
+```json
+{
+   "kind":"Policy",
+   "apiVersion":"v1",
+   "predicates":[
+      {
+         "name":"NoVolumeZoneConflict"
+      },
+      {
+         "name":"MaxEBSVolumeCount"
+      },
+      {
+         "name":"MaxGCEPDVolumeCount"
+      },
+      {
+         "name":"MaxAzureDiskVolumeCount"
+      },
+      {
+         "name":"MatchInterPodAffinity"
+      },
+      {
+         "name":"NoDiskConflict"
+      },
+      {
+         "name":"GeneralPredicates"
+      },
+      {
+         "name":"CheckNodeMemoryPressure"
+      },
+      {
+         "name":"CheckNodeDiskPressure"
+      },
+      {
+         "name":"PodToleratesNodeTaints"
+      },
+      {
+         "name":"CheckVolumeBinding"
+      }
+   ],
+   "priorities":[
+      {
+         "name":"ImageLocalityPriority",
+         "weight":1
+      },
+      {
+         "name":"SelectorSpreadPriority",
+         "weight":1
+      },
+      {
+         "name":"InterPodAffinityPriority",
+         "weight":1
+      },
+      {
+         "name":"LeastRequestedPriority",
+         "weight":1
+      },
+      {
+         "name":"BalancedResourceAllocation",
+         "weight":1
+      },
+      {
+         "name":"NodeAffinityPriority",
+         "weight":1
+      },
+      {
+         "name":"TaintTolerationPriority",
+         "weight":1
+      }
+   ],
+}
+
+```
+
+## Predicates
+
```

Comment:
We should break out this content into a separate reference topic, and figure out how to automate updating it if it isn't a dupe with the existing refdocs. As with the example json ^^, it's not maintainable going forward.

Reply:
Yes, cannot agree more, automate generating and updating will be best. Would you please give some suggestions for moving forward. I will be very glad to work on it.